about.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.njk
+title: About / FAQ
+---
+
+# About
+
+**Package Control <span class="hot">R</span>** is the community-driven package management solution for Sublime Text. It's a front-to-back, top-to-bottom, rewrite of the original [packagecontrol.io](https://packagecontrol.io), managed entirely by the Sublime Text community.
+
+The system consists of several components:
+
+- [This website](https://packages.sublimetext.io), listing publicly available packages for Sublime Text.
+- A [website listing libraries](http://packagecontrol.github.io) (a.k.a. dependencies) available for package authors.
```

Comment:
Well, this page is rather dated and only lists python 3.3 dependencies. It is probably not of much value in a ST4 or even python 3.13 world anymore, now that most libraries are directly shipped via pypi. IMHO, that site is obsolete.

Reply:
I think the intention is to change that as soon as #9 is ready.

Reply:
Saw the FAQ after commenting and already left a comment on #9.

Reply:
FWIW, next release will also provide a _Package Control: List Available Libraries_ command, directly in command palette.

Reply:
I've changed it so it mentions a registry of libraries, and no longer links to the libraries sites. We can cycle back later and improve that bit.